vault/assets/configuration/spec.yaml

```diff
@@ -4,15 +4,30 @@ files:
   options:
   - template: init_config
     options:
-    - template: init_config/http
-    - template: init_config/default
+    - template: init_config/openmetrics
   - template: instances
     options:
+    - name: use_openmetrics
+      description: |
+        Use the latest OpenMetrics implementation for more features, better performance,
+        and more appropriate metric types by default. If running on Agent v6, you must
```

Comment:
```suggestion
        and more metric types by default. If running on Agent v6, you must
```

Reply:
This is correct since the types are now different and more accurate